src/core/Folder.java

```diff
@@ -0,0 +1,33 @@
+package core;
+
+import java.util.List;
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ */
+public class Folder extends Item {
+
+    public Folder (File file) {
+        super(file);
+    }
+
+    /**
+     * List of files and folders that are on a path
+     */
+    public List<String> listFilesAndFolders(String directoryName)
```

Comment:
this is the an example of a how should be the documentation for java methods
/**
 * Returns an Image object that can then be painted on the screen. 
 * The url argument must specify an absolute {@link URL}. The name
 * argument is a specifier that is relative to the url argument. 
 * <p>
 * This method always returns immediately, whether or not the 
 * image exists. When this applet attempts to draw the image on
 * the screen, the data will be loaded. The graphics primitives 
 * that draw the image will incrementally paint on the screen. 
 *
 * @param  url  an absolute URL giving the base location of the image
 * @param  name the location of the image, relative to the url argument
 * @return      the image at the specified URL
 * @see         Image
 */
 public Image getImage(URL url, String name) {
        try {
            return getImage(new URL(url, name));
        } catch (MalformedURLException e) {
            return null;
        }
 }